llvm/utils/TableGen/DXILEmitter.cpp

```diff
@@ -377,10 +454,29 @@ static void emitDXILIntrinsicMap(ArrayRef<DXILOperationDesc> Ops,
   OS << "#ifdef DXIL_OP_INTRINSIC\n";
   OS << "\n";
   for (const auto &Op : Ops) {
-    if (Op.Intrinsic.empty())
+    if (Op.IntrinsicSelects.empty()) {
       continue;
-    OS << "DXIL_OP_INTRINSIC(dxil::OpCode::" << Op.OpName
-       << ", Intrinsic::" << Op.Intrinsic << ")\n";
+    }
+    for (const DXILIntrinsicSelect &MappedIntr : Op.IntrinsicSelects) {
+      OS << "DXIL_OP_INTRINSIC(dxil::OpCode::" << Op.OpName
+         << ", Intrinsic::" << MappedIntr.Intrinsic << ", ";
```

Comment:
My understanding is that the inclusion of the last comma is the difference between the last pr and is what prevents one of the warning types.

I was unsure about the default statement you are referring for the other warning?

Reply:
> My understanding is that the inclusion of the last comma is the difference between the last pr and is what prevents one of the warning types.
That's right.

I made a separate comment about the default warning.